css-borders-4/Overview.bs

```diff
@@ -617,16 +617,26 @@ Initial: none
 Applies to: all elements
 Inherited: no
 Percentages: N/A
-Computed value: either 'none' or a list,
-	each item a pair of offsets (horizontal and vertical) from the element‘s box
-Animation type: by computed value
+Computed value: list, each item either 'none' or a pair of offsets
+  (horizontal and vertical) from the element‘s box
+Animation type: by computed value,
+  treating ''box-shadow-offset/none'' as ''0 0''
+  when interpolated with non-'none' values.
```

Comment:
This needs to say that if the lists have different lengths, `none` values are added to the short list until the have the same length.

Something analogous for the other longhands. Possibly consider a new animation type like "defaultable list" or something, since there already is a "repeatable list".

Reply:
I meant to add the everything related to the coordinating list property group in https://github.com/w3c/csswg-drafts/pull/8613.

I was also thinking about the animation type, as "repeatable list" doesn't fit here. @fantasai Any ideas?